features/junit_format.feature

```diff
@@ -0,0 +1,616 @@
+  Feature: JUnit Formatter
+  In order integrate with other development tools
+  As a developer
+  I need to be able to generate a JUnit-compatible report
+
+  Scenario: Normal Scenario's
+    Given a file named "features/bootstrap/FeatureContext.php" with:
+      """
+      <?php
+
+      use Behat\Behat\Context\CustomSnippetAcceptingContext,
+          Behat\Behat\Tester\Exception\PendingException;
+
+      class FeatureContext implements CustomSnippetAcceptingContext
+      {
+          private $value;
+
+          public static function getAcceptedSnippetType() { return 'regex'; }
+
+          /**
+           * @Given /I have entered (\d+)/
+           */
+          public function iHaveEntered($num) {
+              $this->value = $num;
+          }
+
+          /**
+           * @Then /I must have (\d+)/
+           */
+          public function iMustHave($num) {
+              PHPUnit_Framework_Assert::assertEquals($num, $this->value);
+          }
+
+          /**
+           * @When /I add (\d+)/
+           */
+          public function iAdd($num) {
+              $this->value += $num;
+          }
+
+          /**
+           * @When /^Something not done yet$/
+           */
+          public function somethingNotDoneYet() {
+              throw new PendingException();
+          }
+      }
+      """
+    And a file named "features/World.feature" with:
+      """
+      Feature: World consistency
+        In order to maintain stable behaviors
+        As a features developer
+        I want, that "World" flushes between scenarios
+
+        Background:
+          Given I have entered 10
+
+        Scenario: Undefined
+          Then I must have 10
+          And Something new
+          Then I must have 10
+
+        Scenario: Pending
+          Then I must have 10
+          And Something not done yet
+          Then I must have 10
+
+        Scenario: Failed
+          When I add 4
+          Then I must have 13
+
+        Scenario Outline: Passed & Failed
+          When I add <value>
+          Then I must have <result>
+
+          Examples:
+            | value | result |
+            |  5    | 16     |
+            |  10   | 20     |
+            |  23   | 32     |
+      """
+    And there is a folder named "junit"
```

Comment:
I think this should not be necessary. The formatter should create the folder if it does not exist yet (just like other formatters are creating the file)